content/media.md

```diff
@@ -0,0 +1,19 @@
++++
+title = 'Docs & Media'
+date = 2025-03-28T13:46:05+01:00
+draft = true
++++
+
+## uProtocol motivation
+
+[A motivation and explanation for the uProtocol approach]({{< relref "explainer.md" >}})
```

Comment:
IMHO we should use (a slightly adapted version of) this instead of the existing `/motivation` page ...

Reply:
@AnotherDaniel WDYT?

Reply:
I'll do an attempt to unify these two, yes. Needed to think about it for a bit, but I'll update later today.

Reply:
Alright - I do like the existing motivation text, so I kept it - but added a paragraph at the end and linked to the more fundamental explainer article.